python/ql/src/Expressions/Formatting/AdvancedFormatting.qll

```diff
@@ -1,4 +1,5 @@
 import python
+private import semmle.python.types.Builtins
```

Comment:
Is this used anywhere?

Reply:
oh! good catch. i had used it for a thing then replaced that thing with a different implementation and forgot to remove the import. thanks! :)